nautobot/extras/migrations/0031_tag_content_types.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.1.14 on 2022-03-19 23:21
+
+from django.db import migrations, models
+import nautobot.extras.models.tags
+import nautobot.extras.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("extras", "0030_webhook_alter_unique_together"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="tag",
+            name="content_types",
+            field=models.ManyToManyField(
+                blank=True,
+                limit_choices_to=nautobot.extras.utils.ModelSubclassesQuery(
+                    "nautobot.core.models.generics.PrimaryModel"
+                ),
+                related_name="tags",
+                to="contenttypes.ContentType",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="gitrepository",
+            name="tags",
+            field=nautobot.extras.models.tags.TaggableManagerField(through="extras.TaggedItem", to="extras.Tag"),
+        ),
+        migrations.AlterField(
+            model_name="job",
+            name="tags",
+            field=nautobot.extras.models.tags.TaggableManagerField(through="extras.TaggedItem", to="extras.Tag"),
+        ),
+        migrations.AlterField(
+            model_name="secret",
+            name="tags",
+            field=nautobot.extras.models.tags.TaggableManagerField(through="extras.TaggedItem", to="extras.Tag"),
+        ),
+    ]
```

Comment:
Don't we still need a data migration as well? To set the content_types on all pre-existing Tags to match all allowed content-types, instead of the blank/empty default list?